### ISSUE TITLE
pylint error in beancount.core.data

### ISSUE THREAD

**Issue by blais** [beancount/beancount#343]
**[Original report](https://bitbucket.org/blais/beancount/issue/343) by Johannes Harms (Bitbucket: [johannesjh](https://bitbucket.org/johannesjh), GitHub: [johannesjh](https://github.com/johannesjh)).**

----------------------------------------

A pylint error in beancount.core.data makes problems in smart_importer's continuous integration:

```
************* Module smart_importer.predictor
smart_importer/predictor.py:10:0: E0001: Cannot import 'beancount.core.data' due to syntax error 'misplaced type annotation (<unknown>, line 283)' (syntax-error)
************* Module smart_importer.pipelines
smart_importer/pipelines.py:6:0: E0001: Cannot import 'beancount.core.data' due to syntax error 'misplaced type annotation (<unknown>, line 283)' (syntax-error)
```

see https://travis-ci.org/beancount/smart_importer/jobs/465570874 for full log output.

**Comment by blais** [comment]
**Original comment by Johannes Harms (Bitbucket: [johannesjh](https://bitbucket.org/johannesjh), GitHub: [johannesjh](https://github.com/johannesjh)).**

----------------------------------------

I am currently preparing a merge request for this.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #343 (pylint for Johannes Harms)

**Comment by blais** [comment]
**Original comment by Johannes Harms (Bitbucket: [johannesjh](https://bitbucket.org/johannesjh), GitHub: [johannesjh](https://github.com/johannesjh)).**

----------------------------------------

thank you!
